### PR TITLE
Add extension development instructions for Windows users

### PR DESCRIPTION
**Pull request by nevalla**
Signed-off-by: Lauri Nevala <lauri.nevala@gmail.com>

### CODE REVIEW

File: docs/extensions/get-started/your-first-extension.md [hunk 1]
```diff
@@ -16,12 +16,36 @@ git clone https://github.com/lensapp/lens-extension-samples.git
 
 Next you need to create a symlink from the directory that Lens will monitor for user installed extensions to the sample extension, in this case **helloworld-sample**:
 
+**Linux & MacOS**
 ```sh
 mkdir -p ~/.k8slens/extensions
 cd ~/.k8slens/extensions
 ln -s <lens-extension-samples directory>/helloworld-sample helloworld-sample
```
Comment (stevejr) [inline docs/extensions/get-started/your-first-extension.md]:
```suggestion
ln -s lens-extension-samples/helloworld-sample helloworld-sample
```

File: docs/extensions/get-started/your-first-extension.md [hunk 2]
```diff
@@ -16,12 +16,36 @@ git clone https://github.com/lensapp/lens-extension-samples.git
...
+
+```sh
+mkdir C:\Users\<user>\.k8slens\extensions -force
+cd C:\Users\<user>\.k8slens\extensions
+```
+
+If you have administrator rights, you can create symlink to the sample extension, in this case **helloworld-sample**:
+
+```sh
+cmd /c mklink /D helloworld-sample <lens-extension-samples directory>\helloworld-sample
```
Comment (stevejr) [inline docs/extensions/get-started/your-first-extension.md]:
```suggestion
cmd /c mklink /D helloworld-sample lens-extension-samples\helloworld-sample
```

File: docs/extensions/get-started/your-first-extension.md [hunk 3]
```diff
@@ -16,12 +16,36 @@ git clone https://github.com/lensapp/lens-extension-samples.git
...
+```
+
+Without administrator rights, you need to copy the extensions sample directory into `C:\Users\<user>\.k8slens\extensions`:
+
+```
+Copy-Item '<lens-extension-samples directory>\helloworld-sample' 'C:\Users\<user>\.k8slens\extensions\helloworld-sample'
```
Comment (stevejr) [inline docs/extensions/get-started/your-first-extension.md]:
```suggestion
Copy-Item 'lens-extension-samples\helloworld-sample' 'C:\Users\<user>\.k8slens\extensions\helloworld-sample'
```

**Review by stevejr — COMMENTED** [review]
@nevalla - See suggestions - I think we just reference the  lens-extension-samples location directly?

**Comment by nevalla** [comment]
> @nevalla - See suggestions - I think we just reference the lens-extension-samples location directly?

Yes, that is better.

**Review by stevejr — APPROVED** [review]
LGTM

**Pull request merged**